integral_view/forms/system_date_time_form.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+
+
+class date_time_form(forms.Form):
+    system_time = forms.CharField(required=False)
+    system_date = forms.CharField(required=False)
+
+    def clean(self):
+        cd = super(date_time_form, self).clean()
+        if cd["system_time"] == "" and cd["system_date"] == "":
+            raise forms.ValidationError("Atleast date or time should be present")
```

Comment:
Here, you need to say which field has a problem so that you can display the error next to that particular field using something like self._errors['system_time'] = 'Atleast date or time should be present' and same for system_date. Look at zfs_forms.py's clean..